src/main/java/pt/tqs/g205/resources/IngredienteResource.java

```diff
@@ -0,0 +1,34 @@
+package pt.tqs.g205.resources;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import pt.tqs.g205.domain.Ingrediente;
+import pt.tqs.g205.services.IngredienteService;
+
+import java.util.List;
+
+/**
+ * Controlador REST para expor ingredientes.
+ */
+@RestController
+@RequestMapping(value = "/api/ingredientes")
+public class IngredienteResource {
+
+  @Autowired
+  private IngredienteService ingredienteService;
+  
+  /**
+   * Endpoint para obter todos os ingredientes.
+   * @return todos os ingredientes.
+   */
+  @RequestMapping(value = "", method = RequestMethod.GET)
+  public ResponseEntity<?> getAll() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=6&issues=AWNEkG2aXOBV_Yrh_0kd&open=AWNEkG2aXOBV_Yrh_0kd)

[ISSUE_KEY]: AWNEkG2aXOBV_Yrh_0kd